mgmtsystem/ec2.py

```diff
@@ -467,3 +493,30 @@ def release_address(self, address):
     def get_all_unattached_volumes(self):
         return [volume for volume in self.api.get_all_volumes() if not
                 volume.attach_data.status]
+
+    def delete_sqs_queue(self, queue_id):
+        queue_name = "manageiq-awsconfig-queue-{}".format(queue_id)
```

Comment:
I wanted to make it simplier for calling it from Sprout.
In sprout I can get GUID of EC2 provider, which is that queue_id, so I wanted just call the method with that GUID.
But in this sense I am going to make it not specific for manageiq.